trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
-software and are able to level up their abilities. As a result, a healthy
-community will also be growing.
+https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] tend to stick around, can spend most of their time developing software, and are able to level up their abilities.
+As a result, a healthy community will be continually growing.
 
 Why do https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] join and stick around in a community? Some do because they
 subscribe to the purpose or the mission of the community. It is the Trusted Committer's job to
 clearly articulate and promote this purpose. The importance of this is often
-not recognized - marketing a community and their products is truly essential.
+not recognized, but marketing a community and its products is truly essential.
 
-Another, more obvious reason for people to stick around is that they enjoy
-working with other members of the community, including the Trusted Committers. In my
-experience, what this comes down to is that community members treat and
-communicate with each other with utmost respect. Contributions are treated as
-like a gift or a donation, rather than something that detracts from their own
-work and they laud excellent and especially first contributions. The Trusted Committer's job
-in all this is primarily to set an example for others, similarly to setting an
-example for the level of software quality that is expected.  If necessary, the
-Trusted Committers are the ones who should create and enact a code of conduct for the
-community. Should there be community members whose behavior is detrimental or
-even toxic to the community health, it is the Trusted Committer's responsibility to either
-try and change or contain this or, in the worst case, to remove people from the
-community. Trusted Committers should create opportunities for people to get together
-regularly and get to know each other personally.
+Another, more obvious reason for people to stick around is that they
+enjoy working with other members of the community, including the Trusted Committers. A thriving community is one where members treat
+and communicate with each other with the utmost respect. Contributions are
+treated like gifts or donations, and excellent (especially
+first) contributions are lauded. The Trusted Committer’s job in all this is primarily to set an
+example for others, similar to setting an example for the level of
+software quality expected. If necessary, the Trusted Committers are the ones
+who should create and enact a code of conduct for the community. If
+there are community members whose behavior is detrimental or toxic to
+the community’s health, it is the Trusted Committer’s responsibility to address this. 
```

Comment:
The original text has specific advice on options here (attempt to fix, contain, remove) which the changed text does not have anymore. 
Do we have any rationale for keeping or removing this?

Reply:
Assuming we are talking about the line "it is the Trusted Committer's responsibility to either try and change or contain this or, in the worst case, to remove people from the community."
I actually like O'Reilly version better. 
1 - "try and change or contain this" is neither grammatically clear nor stylistically eloquent
2 - IMHO "to address" gives people freedom to act on the unwanted behavior in the way they find most useful. This article is not a manual in people leadership, and I suggest we do not give prescriptive advise in this area.
3 - probably somewhat repeating what I said in p.1 - I don't know what "try AND change" means...

Reply:
Hm, good points indeed. 
1) yup
2) I'm was hoping for people to have good judgement and not intentionally be dicks...yes, I know. ;)  
3) Good points. Do we leave it as is, e.g. add the options I mentioned in parentheses to the O'Reilly version or just take the O'Reilly version?

Reply:
I'm leaving O'Reilly version.